exampleCourse/questions/element/multipleChoice/question.html

```diff
@@ -260,3 +260,33 @@
     </p>
   </div>
 </div>
+
+<div class="card my-2">
+  <div class="card-header">
+    Part 10
+  </div>
+
+  <div class="card-body">
+
+    <pl-question-panel>
+      <p>
+        Here, the <code>pl-multiple-choice</code> element displays a different placeholder text. This is done  
+        by setting <code>placeholder="Select a color"</code>. You can also have no placeholder by setting it as an empty
+        string, e.g. <code>placeholder=""</code>. If the options are larger than the placeholder text, you
+        may want to set the <code>size</code> to a number large enough to span the options to avoid the text
+        wrapping inside of the dropdown such as <code>size="10"</code> in this example.
```

Comment:
Is specifying `size` here still necessary? After the recent changes, text still doesn't wrap even without a `size` attribute.

I'm questioning if this even needs to be a separate example. The only meaningful difference between this and the one above is the addition of the `placeholder` attribute. Maybe we should just roll that into the above example. What do you think?

Reply:
Happy to remove `size` or roll it into the above example. Just want to double check that adding the parameter isn't going to change any of the open variants? I don't think so but it was something [Eliot pointed](https://github.com/PrairieLearn/PrairieLearn/pull/10723#discussion_r1780364703) out to be careful of in another one of the original changes

Reply:
`size` is used strictly during rendering and isn't persisted anywhere, so this is a completely safe change to make.

Reply:
Sorry, I was referring to the `placeholder`  not `size` (updated it after posting) but I think the same logic applies here. I will go update it now

Reply:
Yep, the exact same logic applies to the code you just wrote.